### ISSUE TITLE
messages aren't saved in the tables - mysql

### ISSUE THREAD

**Issue by uyaharneit** [MetricsGrimoire/MailingListStats#78]
Hi all,

I'm interested to use this tool as one of tools to do my research in natural language processing. And I have already installed everything needed so that this tool can be used to help me to analyse messages in the mailing list Yet, when I tried to retrieve all data from the list, I got this: 

`
0 messages analyzed
0 messages stored in database mlstats
0 messages ignored by the parser
INFO: Everything seems to be ok.
`

I check on the tables in the database and as the messages appeared that no messages at all in the tables.
Anyone can solve this problem. 

Very appreciate for your reply and help in advance.

cheers,
neit

**Comment by jgbarah** [comment]
Hi!

Your description of the problem is strange, since usually MLStats needs just to be pointed to the mailing list archive, and that's it. Maybe the archive is not in the right format?

In any case, I suggest that you give a try to [Perceval](https://github.com/chaoss/grimoirelab-perceval), a part of [GrimoireLab](https://chaoss.github.io/grimoirelab/), described in ["GrimoireLab: A toolset for software development analytics"](https://peerj.com/articles/cs-601/) and [Perceval: software project data at your will](https://dl.acm.org/doi/10.1145/3183440.3183475). They are complete reimplementations of tools in MetricsGrimoire, and are much better maintained these days.

**Comment by uyaharneit** [comment]
``Hi @jgbarah,

Thanks for your reply.
Sorry if the description of the problem is strange. 
Let me make it clearer: I run this command:

`mlstats --db-driver=mysql --db-user=xxx --db-password=xxx --db-host=localhost --db-name=mlstats http://archives.gentoo.org/gentoo-dev/` 

And I got the this:

```
0 messages analyzed 
0 messages stored in database mlstats 
0 messages ignored by the parser 
INFO: Everything seems to be ok
```

I checked on the database, which is mlstats, there is no messages saved . It supposed to be that all data of the mailing list saved in the database, right? But it's otherwise. So, what am I missing here? Is it because the URL of the mailing list? it can't support the gentoo-dev mailing list format?

I will consider to use Perceval and Grimoirelab, anyway. But, I need to fixed this problem first.

Thanks heaps in advance.
